app/public/app.webmanifest

```diff
@@ -0,0 +1,23 @@
+{
+    "lang": "en",
```

Comment:
Maybe you can figure out how we can add a lang="en" attribute to the html element with next aswell, I think Lighthouse is complaining about that aswell

Reply:
Resolved in latest commit.